website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -130,27 +130,30 @@ $ cdktf deploy --help
+  -o, --output                                  Output directory for the synthesized Terraform config                                                                                                                                                             [required] [default: "cdktf.out"]
+      --auto-approve                            Auto approve                                                                                                                                                                                                             [boolean] [default: false]
+      --outputs-file                             Path to file where stack outputs will be written as JSON                                                                                                                                                                                   [string]
+      --outputs-file-include-sensitive-outputs   Whether to include sensitive outputs in the output file                                                                                                                                                                  [boolean] [default: false]
+      --ignore-missing-stack-dependencies       Don't check if all stacks specified in the command have their dependencies included as well                                                                                                                              [boolean] [default: false]
+      --parallelism                             Number of concurrent CDKTF stacks to run. Defaults to infinity, denoted by -1                                                                                                                                                [number] [default: -1]
+      --refresh-only                            Select the "refresh only" planning mode, which checks whether remote objects still match the outcome of the most recent Terraform apply but does not propose any actions to undo any changes made outside of Terraform.  [boolean] [default: false]
+      --terraform-parallelism                   Customize number of parallel graph traversals by Terraform. By default, the parallelism flag is not forwarded to Terraform. Note: This flag is not supported by remote / cloud backend                                       [number] [default: -1]
+  -h, --help                                    Show help                                                                                                                                                                                                                                 [boolean]
+```
+
+~> **Note:** The `parallelism` flag has a different behavior than the [terraform parallelism flag](https://www.terraform.io/cli/commands/apply#parallelism-n). To set the custom terraform parallelism flag, please use the `--terraform-parallelism` flag instead.
```

Comment:
A couple of edits for style. But one thing I actually want to recommend is --> Can we tell users how the behavior is different so they can decide which flag they want to use? We're noting that it IS different, which is great, but if the user isn't super familiar they might need help understanding the best path for them

```suggestion
-> **Note:** The `parallelism` flag has a different behavior than the [Terraform `parallelism` flag](https://www.terraform.io/cli/commands/apply#parallelism-n). Use `--terraform-parallelism` to set the custom terraform parallelism flag.
```